k8s/rabbitmq/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,60 @@
 actions:
-- name: dummy test
+- name: RabbitMQ test
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-rabbitmq-svc.${NAMESPACE}.svc.cluster.local"
+      PORT=15672
+      USERNAME=''
```

Comment:
This all variables at the top of the test will be not necessary, if you move variables to `tester.yaml.template` file.